code/src/App.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react'
+import SurveyForm from 'components/SurveyForm.js'
+
 
 export const App = () => {
+
+
   return (
     <div>
-      Find me in src/app.js!
+      <SurveyForm />
```

Comment:
You followed the instructions on common pitfall by importing only one component into App.js which made it look very clean and ofcourse readable ;)